presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixHdfsInitializer.java

```diff
@@ -26,6 +26,6 @@
 @Retention(RUNTIME)
 @Target({FIELD, PARAMETER, METHOD})
 @Qualifier
-public @interface ForRubix
```

Comment:
I don't see a reason for rename. I find it better the way it was.

Reply:
If the name is the same, it wouldn't be obvious what's the binding for. This is specifically for binding `Hdfs` file system. Other FS are still statically bound in `io.prestosql.plugin.hive.rubix.RubixInitializer#updateRubixConfiguration`

Reply:
> If the name is the same, it wouldn't be obvious what's the binding for

That is correct, but this is why binding requires the class and annotation. Class is the most important part while annotation is just auxiliary.